apps/web/components/LifeEventsSection/LifeEventsSection.tsx

```diff
@@ -1,60 +1,109 @@
 import { GridItems, IconTitleCard } from '@island.is/web/components'
 import { LinkType, useLinkResolver } from '@island.is/web/hooks/useLinkResolver'
-import { GetFrontpageQuery } from '@island.is/web/graphql/schema'
+import { LifeEventPage } from '@island.is/web/graphql/schema'
+import CardWithFeaturedItems from '../CardWithFeaturedItems/CardWithFeaturedItems'
 
 interface LifeEventsSectionProps {
   heading: string
   headingId: string
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore make web strict
-  items: GetFrontpageQuery['getFrontpage']['lifeEvents']
+  seeMoreText: string
+  items: LifeEventPage[]
 }
 
 export const LifeEventsSection = ({
   heading,
   headingId,
   items = [],
+  seeMoreText,
 }: LifeEventsSectionProps) => {
   const { linkResolver } = useLinkResolver()
 
   return (
     <>
       <GridContainer>
-        <Text variant="h3" as="h2" id={headingId}>
-          {heading}
-        </Text>
+        <Box display="flex" flexDirection="row" justifyContent="spaceBetween">
+          <Text variant="h2" as="h2" id={headingId}>
+            {heading}
+          </Text>
+          <Box display={['none', 'none', 'none', 'block']}>
+            <Link href={linkResolver('lifeevents').href} skipTab>
+              <Button
+                icon="arrowForward"
+                iconType="filled"
+                variant="text"
+                as="span"
+              >
+                {seeMoreText}
+              </Button>
+            </Link>
+          </Box>
+        </Box>
       </GridContainer>
       <GridItems
-        mobileItemWidth={270}
-        mobileItemsRows={2}
-        paddingTop={3}
+        mobileItemWidth={215}
+        mobileItemsRows={1}
+        paddingTop={4}
         paddingBottom={3}
         insideGridContainer
+        third
       >
         {items
+          .slice(0, 6)
           .filter((x: { slug: string; title: string }) => x.slug && x.title)
-          .map(
-            (
-              // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-              // @ts-ignore make web strict
-              { __typename: typename, title, shortTitle, slug, tinyThumbnail },
-              index: number,
-            ) => {
-              return (
-                <IconTitleCard
-                  key={index}
-                  heading={shortTitle || title}
-                  imgSrc={tinyThumbnail?.url}
-                  alt={tinyThumbnail?.title}
-                  dataTestId={'lifeevent-card'}
-                  href={linkResolver(typename as LinkType, [slug]).href}
-                />
-              )
-            },
-          )}
+          .map((lifeEvent, index: number) => {
+            return (
+              <CardWithFeaturedItems
+                key={index}
+                heading={lifeEvent.shortTitle || lifeEvent.title}
+                imgSrc={lifeEvent.tinyThumbnail?.url ?? ''}
+                alt={lifeEvent.tinyThumbnail?.title ?? ''}
```

Comment:
The alt text here provides no further context since the images are purely decorative so I suggest setting the alt to an empty string